bootstrap.sh

```diff
@@ -5,18 +5,44 @@ cd $(dirname "$BASH_SOURCE")
 set -ev
 
 # Update puppet to version >= 3.2.2 before using puppet provisioning.
-wget https://apt.puppetlabs.com/puppetlabs-release-pc1-trusty.deb
-dpkg -i puppetlabs-release-pc1-trusty.deb
+package_name=puppetlabs-release-pc1-trusty.deb
```

Comment:
Well... Isn't every constant a variable in a way? I have amended that section to rely on ALL_CAPS :-)